examples/docker/from_anywhere/pipeline.yaml

```diff
@@ -0,0 +1,15 @@
+runtime:
+  container_commands:
+    - "apt update -y"
+    - "apt install -y git"
+    # - "apt-get update && \
+    #   apt-get -y install gcc mono-mcs && \
+    #   rm -rf /var/lib/apt/lists/*"
+  python:
+    python_version: "3.10"
```

Comment:
not related to this PR, but just commenting on this while I'm here: wonder if we can drop the `python_` prefix since it's already in a `python` section, e.g. we could just have:

```
python:
  version: "3.10"
  requirements:
    - "blah==1.0"
```